lldb/include/lldb/Target/Process.h

```diff
@@ -874,10 +874,10 @@ class Process : public std::enable_shared_from_this<Process>,
   /// \see Thread:Resume()
   /// \see Thread:Step()
   /// \see Thread:Suspend()
-  Status Resume();
+  Status Resume(lldb::RunDirection direction = lldb::eRunForward);
```

Comment:
internal APIs, anything not in the `lldb` namespace _can_ be changed. So this is ok. Though I personally would like to see a:
```
Status ReverseResume();
```
I am open to feedback from other here as my mind can easily be changed.

Reply:
The decision was to use a direction flag.